CODING_STYLE.md

```diff
@@ -46,6 +46,8 @@ foo->bar(someLongVariableName,
 cout << "some very long string that contains completely irrelevant text that talks about this and that and contains the words \"lorem\" and \"ipsum\"" << endl;
 ```
 
+To assist in getting the right coding style automatically, the repository contains an [EditorConfig](https://editorconfig.org/) [`.editorconfig`](https://github.com/ethereum/solidity/blob/develop/.editorconfig) file, which describes some of the styles used and which is recognized by many IDE's and editors.
```

Comment:
"assist in getting the right coding style automatically" is a bit of an exaggeration. The file basically only says that we use tabs for C++.

Reply:
And it does say stuff about other types of files too like CMakeList.txt and solidity files. 

See if 38dfd28 addresses your concerns/observations. If not please change as desired. Thanks.